src/components/app/userActionFormVotingInformationResearched/sections/address.tsx

```diff
@@ -0,0 +1,185 @@
+            'Subscribed to notifications': formValues.shouldReceiveNotifications,
+          },
+          payload: { ...formValues, address },
+          onError: (_, e) => {
+            form.setError('root', {
+              message: e.message,
+            })
+            toastGenericError()
+          },
+        },
+        payload =>
+          actionCreateUserActionVotingInformationResearched(payload).then(actionResult => {
+            if (actionResult && 'user' in actionResult && actionResult.user) {
+              identifyUserOnClient(actionResult.user)
+            }
+            return actionResult
+          }),
+      )
+      if (result.status === 'success') {
+        router.refresh()
+        handleTurboVoteRedirect(address)
+        onSuccess(address)
+      }
+    },
+    [form, handleTurboVoteRedirect, onSuccess, router],
+  )
+
+  return (
+    <UserActionFormLayout>
+      <div className="flex h-full flex-1 flex-col">
+        <div className="space-y-6">
+          <p className="text-center text-2xl font-semibold">Prepare to vote</p>
+          <p className="text-center text-fontcolor-muted">
+            Find your polling location and check to see if there are early voting options in your
+            district.
+          </p>
+
+          <div className="mx-auto w-full max-w-lg">
+            <Form {...form}>
+              <form
+                id="view-key-races-form"
+                onSubmit={form.handleSubmit(async formValues => {
+                  await createAction(formValues)
+                }, trackFormSubmissionSyncErrors(FORM_NAME))}
+              >
+                <FormField
+                  control={form.control}
+                  name="address"
+                  render={({ field }) => (
+                    <FormItem>
+                      <FormControl aria-invalid={!!error}>
+                        <GooglePlacesSelect
+                          {...field}
+                          className="rounded-full bg-secondary"
+                          onChange={newAddress => {
+                            form.clearErrors('root')
+                            field.onChange(newAddress)
+                          }}
+                          placeholder="Your full address"
+                          ref={inputRef}
+                          value={field.value}
+                          variant="lg"
+                        />
+                      </FormControl>
+                      {!!error && <ErrorMessage>{error?.message}</ErrorMessage>}
+                    </FormItem>
+                  )}
+                />
+              </form>
+            </Form>
+          </div>
+        </div>
+
+        <Button
+          className="mx-auto mt-auto"
+          disabled={form.formState.isSubmitting || !addressField?.place_id}
+          form="view-key-races-form"
+          size="lg"
+          type="submit"
+        >
+          Check your voting information
```

Comment:
In other forms we usually show a `Loading...` message when we're submitting (not the prettiest, but effective), let's add it here too
```suggestion
          {form.formState.isSubmitting ? 'Loading...' : 'Check your voting information'}
```